clean_project/project/search_engine.py

```diff
@@ -21,11 +23,50 @@ def search(self, search_word):
         sslmode = load_config_file(self.connect_path).get('sslmode')
         connection = psycopg2.connect(host=host, port=port, sslmode=sslmode,
                                       database=database, user=user, password=password)
+        words = tuple(search_word.split(" "))
+        count_words = len(words)
+        if len(words) == 1:
+            words += ("",)
```

Comment:
It's necessary because if you have only one word, tuple add automatically the comma after it.

Example: tuple =  ("word",)